db/migrate/20170913210806_change_overridden_by_to_overridden_by_id.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeOverriddenByToOverriddenById < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :doctors_notes, :overridden_by, :overridden_by_id
```

Comment:
I disagree with this change. It's pretty common to have `_by` columns which point to users. Instead, you should tell the model that it's a foreign key. There are examples in our other apps.